classes/Meta.php

```diff
@@ -387,15 +387,10 @@ public static function getCategoryMetas($idCategory, $idLang, $pageName, $title
                     $row['meta_description'] = strip_tags($row['description']);
                 }
 
-                // Paginate title
-                if (!empty($row['meta_title'])) {
-                    $row['meta_title'] = $title . $row['meta_title'] . (!empty($pageNumber) ? ' (' . $pageNumber . ')' : '');
-                } else {
-                    $row['meta_title'] = $row['name'] . (!empty($pageNumber) ? ' (' . $pageNumber . ')' : '');
-                }
-
                 if (!empty($title)) {
-                    $row['meta_title'] = $title . (!empty($pageNumber) ? ' (' . $pageNumber . ')' : '');
+                    $row['meta_title'] = $title;
+                } else {
+                    $row['meta_title'] = $row['meta_title'] ?: $row['name'];
                 }
```

Comment:
```suggestion
                $row['meta_title'] = $title ?: $row['meta_title'] ?: $row['name'];
```

Reply:
Please note if `$title = '0'`, then the value is dismissed here.
Such weird title is very unlikely, but that's an edge case.

Reply:
That edge case wasn't taken into account before. 
If we want to take it into account now, we can do it like that:
               if ($title != '') {
                    $row['meta_title'] = $title;
                } else {
                    $row['meta_title'] = $row['meta_title'] ?: $row['name'];
                }
Writing same conditions in one line will be not easily readable